lib/mumukit/flow/suggesting.rb

```diff
@@ -0,0 +1,38 @@
+module Mumukit::Flow
+  module Suggesting
+    def next_suggested_item_for(submitter)
+      set_adaptive_assignment!(submitter)
+      next_item_suggestion(submitter).item
+    end
+
+    private
+
+    def next_item_suggestion(submitter)
+      if end_reached?
+        if pending_siblings?
+          Mumukit::Flow::Suggestion::Revisit.new(first_pending_sibling)
+        else
+          Mumukit::Flow::Suggestion::None
```

Comment:
I'd like to close this before implementing full guide skipping, as I would also have to add many tests and probably change a few other things along the way.

I've made an issue: #3

Reply:
Do note that even without guide skipping it's important to further delegate to the parent since an exercise from the following guide should be suggested when a guide is finished.

Reply:
The guide-to-guide navigation works already because I tried to keep the changes in Laboratory to a minimum. [See line 14](https://github.com/mumuki/mumuki-laboratory/commit/5451e89748ac4f9070c43b6f8d4746a69ab9ee77#diff-0d5e531a8b71817cb070e25dbd0161aaR14), which works almost like it did before: if there's no next exercise, then it's time for the next lesson.

That line in Laboratory should change once guide skipping is possible, since `AdaptiveNavigation.new(self).button(exercise)` should never be nil anyway (unless it's the very last exercise, I guess)